composer.json

```diff
@@ -24,7 +24,8 @@
         "kriswallsmith/assetic": "1.1.2",
         "components/highlightjs": "^9.7",
         "algolia/algoliasearch-client-php": "~1.25.0",
-        "gregwar/rst": "master"
+        "gregwar/rst": "master",
```

Comment:
Do we still rely on `master` here btw?

Reply:
Yes, unfortunately. The library is abandoned as far as I can tell so we are using our fork here. https://github.com/doctrine/doctrine-website/blob/master/composer.json#L18

I plan to work on the library more and clean it up, use our coding standards, run phpstan against it, etc.

Reply:
Are we going to adapt the API of the fork in favour of the doctrine coding style? Some inherit methods need type hints or at least docs.